x-pack/plugins/ingest_pipelines/public/application/components/pipeline_editor/components/shared/map_processor_type_to_form.tsx

```diff
@@ -258,7 +258,10 @@ export const mapProcessorTypeToDescriptor: MapProcessorTypeToDescriptor = {
     }),
     getDefaultDescription: ({ field }) =>
       i18n.translate('xpack.ingestPipelines.processors.defaultDescription.dot_expander', {
-        defaultMessage: 'Expands "{field}" into an object field',
+        defaultMessage:
```

Comment:
I don't think we can have a conditional within the `defaultMessage`, as it may cause issues for translation. Instead, we need 2 separate `i18n.translate()` calls.

```
    getDefaultDescription: ({ field }) => {
      return field === '*'
        ? i18n.translate(
            'xpack.ingestPipelines.processors.defaultDescription.dotExpanderWildcard',
            {
              defaultMessage: 'All top-level fields will be expanded',
            }
          )
        : i18n.translate('xpack.ingestPipelines.processors.defaultDescription.dotExpander', {
            defaultMessage: 'Expands "{field}" into an object field',
            values: {
              field,
            },
          });
    },
```

Reply:
Thanks a lot for the review!!

The related commit is [d6e1087](https://github.com/elastic/kibana/pull/123522/commits/d6e1087df5a43e4929291e84cc36f04f5baa8064)

I have also updated the test case because the error message was amended: [5c0c25f](https://github.com/elastic/kibana/pull/123522/commits/5c0c25fc3c0ed602d7f6fca4cd99a013c4b19b1c)